images/linux/scripts/installers/1604/powershellcore.sh

```diff
@@ -8,8 +8,11 @@
 source $HELPER_SCRIPTS/document.sh
 
 LSB_RELEASE=$(lsb_release -rs)
+#echo "Installing the libicu60 library"
+#wget http://security.ubuntu.com/ubuntu/pool/main/i/icu/libicu60_60.2-3ubuntu3_amd64.deb
+#dpkg -i libicu60_60.2-3ubuntu3_amd64.deb
```

Comment:
Does it really make sense to leave commented code here? Let's just remove it if we cannot see any good reasons.